Jenkinsfile_CNP

```diff
@@ -7,6 +7,7 @@ def product = "data-extractor"
 def component = "job"
 
 withPipeline(type, product, component) {
+  env.TESTCONTAINERS_RYUK_DISABLED = 'true'
```

Comment:
what's this do?

Reply:
> ryuk image helps to remove and cleanup containers.

apparently, what's wrong with it?

Reply:
We have an issue while using test containers on Jenkins, we need to authorise quay repository for ryuk. We were just following some recommendations on issue to test if it resolves. https://github.com/testcontainers/testcontainers-java/issues/700 , apparently it doesn't as we need to authorise basic postgres one as well.

Reply:
weird as lots of apps here use testcontainers on jenkins

Reply:
I think this repo does some manual init of test containers in a rule:
https://github.com/hmcts/cmc-claim-store

may be worth having a look, it specifies which image to use I believe